tools/transformer/src/engine/readers.js

```diff
@@ -141,6 +142,13 @@ function getObjectFromYamlPath(path, obj) {
   return currObj;
 }
 
+function isInlineModel(path) {
+  const pathStack = path.split(".").reverse();
+  pathStack.pop();
+
+  return !(pathStack.pop() == "components" && pathStack.pop() == "schemas");
+}
```

Comment:
Is there any possibility of encountering a path with fewer than 2 segments?

Reply:
Under the `components` object, it should always have one more extra segment. But I will add some more validation for this.